centinel/primitives/dnslib.py

```diff
@@ -158,7 +158,6 @@ def lookup_domain(self, domain, nameserver=None, log_prefix=''):
         results = {'domain': domain, 'nameserver': nameserver}
         # construct the socket to use
         sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
-        sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
```

Comment:
socket.SOL_SOCKET indicates the level of the parameter being set. In this case, socket.SO_REUSEADDR is a socket.SOL_SOCKET level parameter. This whole line means setting socket.SO_REUSEADDR to 1.